### DISCUSSION TITLE
How to exclude folder in public folder from precacheEntries ?

### DISCUSSION THREAD

**Discussion by codefln** [serwist/serwist#109] [General]
I have a folder with sounds. (/sounds/fileName.mp3) What is the correct way to remove it from precacheEntries?

**Comment by DuCanhGH** [comment]
@codefln try using `exclude`.

If this file is in the public/ directory, and you are using `@serwist/next`, try setting `additionalPrecacheEntries` to `[]`. This removes every file in that directory from the precache manifest.

**Reply by codefln** [reply]
Yes, the additionalPrecacheEntries parameter worked. It would be great to add this to the documentation! Thank you very much
